src/frontend/src/components/common/Callout/_callout.scss

```diff
@@ -26,20 +14,15 @@
   justify-content: start;
   background-color: $ntrl-lightgray;
   border-left: 10px solid $primary-colour;
-  width: 1088px;
-  height: 76px;
-  padding: 25px;
+  height: 2.5rem;
+  padding: 40px 25px;
```

Comment:
Could we please use `em`'s here? The conversions are: 2.5em and 1.56em, respectively.